rotkehlchen/chain/ethereum/modules/aave/aave.py

```diff
@@ -74,33 +78,63 @@ def __init__(
         self.history_lock = Semaphore()
         self.balances_lock = Semaphore()
         self.counterparties = [CPT_AAVE_V1, CPT_AAVE_V2]
+        self.v3_data_provider = self.ethereum.contracts.contract(AAVE_V3_DATA_PROVIDER)
+        self.v3_reserve_cache: dict[EvmToken, ReserveData] = {}
+
+    def _get_v3_reserve_data(self, reserve_token: 'EvmToken') -> 'ReserveData | None':
```

Comment:
Aren't these constant once queried? Why save them in memory and not global DB cache?

Reply:
They can be stored in globalDb cache to make it faster. But I could not find anywhere, how frequently they are updated. If they are updated very frequently then better to query every time.